README.md

```diff
@@ -0,0 +1,46 @@
+## What is Novus
+Novus is an MMO Engine.
+
+Currently Novus is developing a game/server solution where our main focus is to be feature complete with the most popular MMOs out there.
+Long term, we hope to support many more features.
+
+## The Novus Promise
+The project was made with the promise that we would always focus on reliability, redundancy, and performance over convenience. We achieve this through making use of experience, but also applying modern techniques and design patterns.
+
+The end-goal is to provide a game/server setup, capable of tackling all of the limitations set by the current standard. Solving those issues are complicated, but we start by applying a proper foundation for our architecture to allow for better flow of information(data) and performance.
+
+## Novus Discord
+The project has an official [Discord](https://discord.gg/gz6FMZa).
+You will find the developers to be active on the discord and always up for answering any questions you might have regarding the project. Despite Novus not currently being ready for production level use, we are always welcoming any users that want to try using it.
+
+## Dependencies
+* [OpenSSL 1.1.0](https://www.openssl.org/source/)
+* [Premake5](https://premake.github.io/)
+* [Vulkan 1.1 (or higher)](https://vulkan.lunarg.com/)
+
+## Libraries
+Here we include a honorable mention to all the libraries included directly into the source of Novus. You do not need to download these on your own, but we felt it was important to address these as without them, Novus would be a lot more time consuming to develop.
+* [Moodycamel ConcurrentQueue](https://github.com/cameron314/concurrentqueue)
+* [Nlohmann Json](https://github.com/nlohmann/json)
+* [Robin Hood Hashing](https://github.com/martinus/robin-hood-hashing)
+* [Amy](https://github.com/liancheng/amy)
+* [Asio](http://sourceforge.net/projects/asio/files/asio/1.12.1%20%28Stable%29/)
+* [Tracy Profiler](https://bitbucket.org/wolfpld/tracy/src/master)
+* [Entt ECS](https://github.com/skypjack/entt/)
+* [Cpp-Taskflow](https://github.com/cpp-taskflow/cpp-taskflow)
+
+## How to build (Microsoft Visual 2022)
+>[!TIP]
+> If you plan to use [symlink](https://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html) to link engine to the [AssetConverter](https://github.com/novusengine/AssetConverter) and [Game](https://github.com/novusengine/Game) projects, you do not need to run premake or build the solution from this folder.
```

Comment:
We actually do not recommend the workflow of symlinking a locally cloned Engine repo in the Submodule folder of game.

It is convenient for development, but you lose the ability to update the submodule and push that to the Game/Assetconverter repo. Now you have to symlink it regularly after updating the submodule. It probably works, but I think this could lead to issues later on so I don't think it should be recommended.